README.md

```diff
@@ -5,35 +5,55 @@ This repo contains code used to run the Monash Human Power camera system. The `m
 
 ## Installation
 
-Firstly, you'll need to create a `.env` file containing either `MHP_CAMERA=primary` or `MHP_CAMERA=secondary`. Similarly, set `MHP_BIKE` to `V2` or `V3` so that the overlays listen for data on the correct MQTT topics. This step may be skipped (and can be overriden with terminal arguments) but should always be done on the actual camera overlays.
+1. **Setup `.env` file**
```

Comment:
omg i didn't realise how much of a difference this would make, but this is SO much clearer.